src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import propTypes from 'prop-types';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} key={movie.imdbId} />
+    ))}
+  </div>
 );
+
+MoviesList.propTypes = {
+  movies: propTypes.arrayOf(propTypes.object),
```

Comment:
use MovieCardShape here

```suggestion
  movies: propTypes.arrayOf(MovieCardShape),
```